#define the goal, srs_msgs/srs_action[] action_sequence	#Specify what action sequence to be applied
string action         # to deprecate, replaced by json_parameters. Kept for backward compatibility
string parameter      # to deprecate, replaced by json_parameters
uint32 priority       # to deprecate
string json_parameters   # new added parameter in json (test by Ze)
# string[] parameters   # to deprecate
---
#define the result
uint32 return_value
---
#define a feedback message
string current_state         # to deprecate
bool solution_required       # to deprecate
uint32 exceptional_case_id   # to deprecate
string json_feedback         # new added feedback in json (test by ze)